src/App.tsx

```diff
@@ -17,18 +49,43 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                query={query}
+                setQuery={setQuery}
+                todosStatus={todosStatus}
+                setTodosStatus={setTodosStatus}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {todos.length > 0
```

Comment:
move to a constant like 
```suggestion
              {isHasTodos
```